builtin/logical/pki/path_tidy.go

```diff
@@ -8,7 +8,6 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/armon/go-metrics"
```

Comment:
I think we are using this import and can't delete it. (See line 117, for example.)